README.md

```diff
@@ -280,14 +281,14 @@ domain = env.furl("DOMAIN")  # => furl('https://myapp.com')
 
 
 # Custom parsers can take extra keyword arguments
-@env.parser_for("enum")
-def enum_parser(value, choices):
+@env.parser_for("choice")
+def choice_parser(value, choices):
```

Comment:
thought (non-blocking): We might want to come up with a different example here since users would use `env.enum` for this use case rather than a custom parser.

Reply:
I think the example isn't that bad for now, this deserves a separate issue anyway.